index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <!-- CSS only -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/styles.css">
+  <script defer src="https://use.fontawesome.com/releases/v5.15.4/js/all.js"
+    integrity="sha384-rOA1PnstxnOBLzCLMcre8ybwbTmemjzdNlILg8O7z1lUkLXozs4DHonlDtnE7fpc"
+    crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <!-- Add NavBar -->
+  <nav class="navbar navbar-expand-lg  fixed-top" style="background-color:#EEEEEE ;">
+    <div class="container-fluid">
+      <img src="assets/music.png" alt="logo" class="img-fluid logo col-4 mx-5">
+      <div class="container col-3"></div>
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0 col-4 ">
+        <li class="nav-item col-4">
+          <a class="nav-link text-dark" aria-current="page" href="#">Features</a>
+        </li>
+        <li class="nav-item col-4">
+          <a class="nav-link text-dark" href="#">How It Works</a>
+        </li>
+        <li class="nav-item col-4">
+          <a class="nav-link text-dark">Pricing</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <!-- get started section  -->
+  <div class="form-container my-5 col-6 text-center">
+    <h4 class="text-center mt-5">Handcrafted, home-made masterpieces</h4>
+    <form class="form-inline mx-5" style="width:20rem ;">
+      <input class="" type="search" placeholder="Enter your email address" aria-label="Search">
+      <button class="btn" type="submit">Get Started</button>
+    </form>
+  </div>
+
+  <!-- Way of operation section -->
+  <div class="main m-auto">
+    <div class="info-section">
+      <div class="icons col-2">
+        <i class="fas fa-fire"></i>
+      </div>
+      <div class="col-6">
+        <h4>
+          Premium Materials</h4>
+        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your
+          purchase.</p>
+      </div>
+    </div>
+
+    <div class="info-section">
+      <div class="icons col-2">
+        <i class="fas fa-truck col"></i>
+      </div>
+      <div class="col-6">
+        <h4>
+          Fast Shipping</h4>
+        <p>We make sure you receive your trombone as soon as we have finished making it. We also provide free returns if
+          you are not satisfied.</p>
+      </div>
```

Comment:
- [ ]  Please 🙏🏽 always explain why you request a change - it might be obvious for you but not for someone else.
- [ ]  Please 🙏🏽 include emojis and gifs to make your code review more human-friendly 👽.